readme.md

```diff
@@ -45,3 +45,5 @@ use new logo
 update auth flow
 
 more documentation
+
+add new user settings table
```

Comment:
thanks for adding a trigger to auto-update the `updated_at` field @botany-lead 💯

Reply:
no problem 👍